index.markdown

```diff
@@ -636,16 +636,18 @@ If you are using [RBC](http://www.rbcroyalbank.com/), you can partner up with th
 RBC, like TD, does transfers for free between countries (NOTE: USD --> USD only!). RBC Bank in the US is purely online - 
 the fee for a basic account is $2.95/month, but this is easily well-worth it for no-hassle, instantaneous transfers of USD between your US and Canadian RBC accounts.
 You can use any PNC Bank ATM in the US to withdraw (USD) money from your RBC account at no fee, as well as a multitude of other ATMs across the country. 
-Transfers between the US and Canadian accounts are unlimited and instantaneous - RBC Bank in the US even accepts Direct Deposits and cheque deposits via mobile service!
+Transfers between the US and Canadian accounts are unlimited and instantaneous - RBC Bank in the US even accepts Direct Deposits and cheque deposits via mobile service (taking a picture of the front/backside of the cheque)!
 The USD account in Canada should be free if you already have an existing service with them; a good rule of thumb is that it may not be worth it to sign up if you aren't already using them.
+Signing up for two things on the Canadian side should keep you from paying anything extra, while validating your RBC US account: "Day-to-Day Chequing" account, and the "no annual fee" credit card from RBC Royal Bank.
 
 Once the USD funds are in your Canadian RBC account, feel free to use XE trade or some other mechanism to convert it to CAD without losing too much in the exchange. 
-As recently as Sept. 2014, Michael got 1.0975 USD/CAD exchanged ($1000 USD).
+When using "Debit My Account" on XE Trade (from a US account), then "ACH/EFT" (to deposit to a Canadian institution, in CAD), you don't have to pay any fees other than a brokerage fee that XE administers as part of the exchange rate.
+As recently as March 2015, Michael got 1.2647 USD/CAD exchanged ($1600 USD).
 
 #### Not Recommended Ways
 
-* Wire transfer (mad fees)
-* Cash a check (takes 20-30 days for the first I hear). Keep in mind, bank currency rates are generally terrible.
+* Wire transfer (mad fees - $40 per transaction, at minimum!)
+* Cash a check (takes 20-30 days for the first I hear). Keep in mind, bank currency rates (are generally terrible.
```

Comment:
I'll make an edit so they appear on the same line, so as to remove confusion. One sec!